### PR TITLE
feat:reducing suppression CVE count.

### PR DESCRIPTION
**Pull request by KOTLAPRA23**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-1745

### Change description ###

feat:reducing suppression CVE count.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -66,117 +63,122 @@
...
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-resource-server-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-resource\-server@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
```
Comment (pksaxena01) [inline config/owasp/suppressions.xml:158]: I hope, you will remove the commented lines before you merge to master.

Comment (KOTLAPRA23) [reply config/owasp/suppressions.xml:158]: yes

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -38,7 +35,7 @@
 <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
 <vulnerabilityName>CVE-2020-9484</vulnerabilityName>
```
Comment (abhijitdiwan) [inline config/owasp/suppressions.xml:36]: we can replace tomcat I think. i did that

File: config/owasp/suppressions.xml [hunk 3]
```diff
@@ -38,7 +35,7 @@
 <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
```
Comment (abhijitdiwan) [inline config/owasp/suppressions.xml:35]: we can remove i think sling and Tomcat.

File: config/owasp/suppressions.xml (hunk 2 above)
Comment (KOTLAPRA23) [reply config/owasp/suppressions.xml:36]: yes ..i just testing ..not final one

**Pull request closed without merging**